README.md

```diff
@@ -115,6 +115,10 @@ python3 chirp.py -p registry yara -t c:\\target_dir\\** -o chirp_result -l debug
            [+] Done! Your results can be found at Z:\README\output.
 ```
 
+## Non-interactive Mode
```

Comment:
This should be triple hashed, as we use the double hash for section headers of our README